### PR TITLE
perlPackages.CiscoIPPhone: init at 0.05

### PR DESCRIPTION
**Pull request by aanderse**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by aanderse** [comment]
@GrahamcOfBorg build perlPackages.CiscoIPPhone perldevelPackages.CiscoIPPhone

File: pkgs/top-level/perl-packages.nix [hunk 1]
```diff
@@ -2322,6 +2322,18 @@ let
...
 
+  CiscoIPPhone = buildPerlPackage {
+    pname = "Cisco-IPPhone";
+    version = "0.05";
+    src = fetchurl {
+      url = "mirror://cpan/authors/id/M/MR/MRPALMER/Cisco-IPPhone-0.05.tar.gz";
+      sha256 = "b03ca263f8f41a6ec545c5393213a3146d36bd45335ade99af51dd42ab6ee16d";
+    };
+    meta = {
+      license = "unknown";
```
Comment (stigtsp) [inline pkgs/top-level/perl-packages.nix:2333]:
```suggestion
      description = "Package for creating Cisco IPPhone XML objects";
      license = with stdenv.lib.licenses; [ artistic1 ];
```

**Review by stigtsp — APPROVED** [review]
Looks good to me, did some basic manual testing of the eaxmples (since the module has a very limited test suite).

**Comment by aanderse** [comment]
Thanks @stigtsp, much appreciated!

**Pull request merged**